File: 30DaysOfCode/day01_dataTypes/dataTypes.c
#include <stdio.h>
#include <string.h>
#include <math.h>
#include <stdlib.h>

int main() {
    int i = 4;
    double d = 4.0;
    char s[] = "HackerRank ";
    
    int another_i;
    double another_d;
    char another_s[100];
    
    scanf("%d %lf %[^\n]s", &another_i, &another_d, another_s);

    printf("%d\n", (i + another_i));
    printf("%.1lf\n", (d + another_d));
    printf("%s%s\n", s, another_s);
    
    return 0;
}

File: 30DaysOfCode/day01_dataTypes/dataTypes.java
import java.io.*;
import java.util.*;
import java.text.*;
import java.math.*;
import java.util.regex.*;

public class Solution {
    public static void main(String[] args) {
        int i = 4;
        double d = 4.0;
        String s = "HackerRank ";
		
        Scanner scan = new Scanner(System.in);
        
        int another_i;
        double another_d;
        String another_s;

        another_i = scan.nextInt();
        another_d = scan.nextDouble();
        scan.nextLine();
        another_s = scan.nextLine();

        System.out.println(i + another_i);
	System.out.println(d + another_d);
        System.out.println(s + another_s);
        
        scan.close();
    }
}
